data/tools/extract-wikidata.py

```diff
@@ -0,0 +1,379 @@
+  'Q78191294': 'singular-possessive',
+  'Q78191289': 'plural-possessive',
+  'Q147276': 'proper-noun',
+  'Q107614077': 'combining-form',
+  'Q956030': 'indefinite-pronoun',
+  'Q53997851': 'definite',
+  'Q53997857': 'indefinite',
+  'Q1941737': 'modal-adverb',
+  'Q12274060': 'future-in-the-past',
+  'Q1329258': 'cardinal-numeral',
+  'Q1775415': 'feminine',
+  'Q499327': 'masculine',
+  'Q1923028': 'gerund',
+  'Q1240211': 'present-perfect',
+  'Q901711': 'colloquial-language',
+  'Q54310231': 'interrogative-pronoun',
+  'Q1522423': 'locative-adverb',
+  'Q1984758': 'misspelling',
+  'Q117605926': 'perfective-verbal-expression',
+  'Q65048529': 'lowercase-text',
+  'Q12812139': 'technical-term',
+  'Q126729404': 'deverbal',
+  'Q131431824': 'proper-verb',
+  'Q113076880': 'postpositive-adverb',
+  'Q130270424': 'interrogative-expression',
+  'Q2034977': 'prepositional-adverb',
+  'Q5978303': 'adverbial-locution',
+  'Q65280376': 'collocation',
+  'Q106610283': 'adverbial-suffix',
+  'Q65248545': 'superlative-adverb',
+  'Q1774805': 'verb',
+  'Q113198319': 'adverbial-particle',
+  'Q2034977': 'prepositional-adverb',
+  'Q3517796': 'simple-tense',
+  'Q146078': 'accusative',
+  'Q145599': 'dative',
+  'Q1317831': 'active',
+  'Q1775461': 'neuter',
+  'Q7977953': 'weak-inflection',
+  'Q3481903': 'strong-inflection',
+  'Q96407524': 'mixed-inflection',
+  'Q55685962': 'subjunctive',
+  'Q54671845': 'subjunctive-2',
+  'Q101583900': 'alternative-spelling',
+  'Q55228835': 'colloquial',
+  'Q100952920': 'zu-infinitive',
+  'Q623742': 'pluperfect',
+  'Q371427': 'imperfective',
+  'Q625420': 'perfect',
+  'Q54792077': 'prefixoid',
+  'Q358417': 'circumposition',
+  'Q34793275': 'demonstrative-pronoun',
+  'Q1050744': 'relative-pronoun',
+  'Q77768790': 'formal',
+  'Q7188068': 'phrasal-template',
+  'Q52434245': 'present-infinitive',
+  'Q15841920': 'idiom',
+  'Q55698475': 'statal-passive',
+  'Q55698471': 'processual-passive',
+  'Q1424306': 'perfective',
+  'Q210523': 'word-stem',
+  'Q202142': 'locative',
+  'Q192997': 'instrumental',
+  'Q1194697': 'passive',
+  'Q53998049': 'indefinite',
+  'Q51927507': 'animate',
+  'Q51927539': 'inanimate',
+  'Q108524486': 'imperfect',
+  'Q216497': 'aorist',
+  'Q12721117': 'continuous',
+  'Q12547192': 'past-imperfect',
+  'Q545779': 'pejorative',
+  'Q109267112': 'affirmative',
+  'Q23199274': 'affirmation',
+  'Q1234617': 'future-perfect',
+  'Q923933': 'ordinal',
+  'Q904896': 'transgressive',
+  'Q114752182': 'preconsonantal',
+  'Q109371067': 'emphatic',
+  'Q113289507': 'emphatic',
+  'Q3502544': 'past-subjunctive',
+  'Q3502553': 'present-subjunctive',
+  'Q23663136': 'past-perfect',
+  'Q2006180': 'pro-form',
+}
+name2qid = {v: k for k, v in qid2name.items()}
+
+filterpos = [
+  'Q187931', 'Q184511', 'Q9788', 'Q30515', 'Q35102',
+  'Q1401131', 'Q5', 'Q80071', 'Q191494', 'Q182990',
+  'Q43229', 'Q41796', 'Q43249', 'Q5456361',
+  'Q3438770',
+]
+
+dictionary = {}
+
+languagecode = 'en'
```

Comment:
This is current best practice on parsing command line args in Python - https://docs.python.org/3/library/argparse.html.

I can fix that once you merge (or you can in this round). Let me know.

Reply:
Here's an example of how to fix it:

```
import argparse  

def main():
  parser = argparse.ArgumentParser(description="Process a language code.")
  parser.add_argument("language_code", nargs="?", default="en", help="The language code (e.g., 'en', 'es', 'fr'). Defaults to 'en'.")
  process_language(args.language_code)

if __name__ == "__main__":
  main()  
```

The __name__ = "__main__": part helps when this is a program run from command line vs being imported as a module - it allows both ways of using it. I can refactor once you merge - important part is arg parsing.